api-docs.templ

```diff
@@ -0,0 +1,81 @@
+# API Documentation
+<a name="top"></a>
+
+{{- range .Files}}
+{{$file_name := .Name}}
+<a name="{{.Name | anchor}}"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## {{.Name}}
```

Comment:
Breaking this doc by filenames of .proto files. I'm not sure anyone would _care_, for instance, that the services, messages, etc. are defined in a file named `api/service/v1alpha1/service.proto`. It may be more important that we clearly demarcate the boundaries of whole APIs. Fortunately, we can probably get away with using the .proto filenames as proxies for whole APIs. Can we maybe just put logic in the template to translate filenames like `api/service/v1alpha1/service.proto` to nicer API names like `service/v1alpha1`? Or something like that?